contents/blog/posthog-changelog.md

```diff
@@ -19,10 +19,41 @@ Interested in what new features, fixes and updates have just launched on PostHog
 
 > Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
 
+## March 10, 2023 
+
+### Dashboard templates
+![dashboard templates](../images/blog/array/dashboard-template-wizard.png)
+
+We want to make it easier and faster for users to find useful information in PostHog — and dashboards are a key part of that because they’re often one of the first things that users build. So, we’ve added a new selection of dashboard templates, as well as a new wizard that’s a little easier on the eye. 
```

Comment:
```suggestion
We want to make it easier and faster for users to find useful information in PostHog — dashboards are a key part of that because they’re often one of the first things users build. So, we’ve added a new selection of dashboard templates, as well as a new wizard that’s a little easier on the eye. 
```